pyproject.toml

```
@@ -1,170 +1,155 @@
-[project]
+[tool.poetry]
 name = "education-backend"
 version = "2022.10.31"
-requires-python = ">=3.10"
-dependencies = [
-    "Django",
-    "django-behaviors",
-    "django-cors-headers",
-    "django-environ",
-    "django-ipware",
-    "django-split-settings",
-    "django-tree-queries",
-    "django-cachalot",
-    "django-prettyjson",
-    "whitenoise",
-    "psycopg2-binary",
-    "shortuuid",
-    "retry",
-    "bcrypt",
-    "simplejson",
-    "Pillow",
-
-    "djangorestframework",
-    "django-storages",
-    "drf-recaptcha",
-    "drf-spectacular[sidecar]",
-    "drf-jwt",
-    "django-filter",
-    "django-anymail",
-    "django-axes",
-    "django-debug-toolbar",
-    "dj-rest-auth",
-
-    "django-healthchecks",
-
-    "cmarkgfm",
-    "bleach",
-
-    "sentry-sdk",
-
-    "babel",
-    "boto3",
-    "PyJWT",
-    "urllib3",
-
-    "celery",
-    "redis",
-
-    "httpx[http2]",
-    "stripe",
-
-    "emoji",
-]
-
-
-[project.optional-dependencies]
-dev = [
-    "ipython",
-
-    "pytest-django",
-    "pytest-cov",
-    "pytest-deadfixtures",
-    "pytest-env",
-    "pytest-freezegun",
-    "pytest-mock",
-    "pytest-randomly",
-    "pytest-repeat",
-    "pytest-xdist",
-    "respx",
-
-    "freezegun",
-    "mixer",
-
-    "black",
-
-    "jedi",
-    "autoflake",
-    "flake8-absolute-import",
-    "flake8-black",
-    "flake8-bugbear",
-    "flake8-cognitive-complexity",
-    "flake8-django",
-    "flake8-eradicate",
-    "flake8-fixme",
-    "flake8-isort>=4.0.0",
-    "flake8-multiline-containers",
-    "flake8-mutable",
-    "flake8-pie",
-    "flake8-pep3101",
-    "flake8-print",
-    "flake8-printf-formatting",
-    "flake8-pytest",
-    "flake8-pytest-style",
-    "flake8-simplify",
-    "flake8-tidy-imports",
-    "flake8-todo",
-    "flake8-variables-names",
-    "flake8-walrus",
-    "flake8-use-fstring",
-    "flake8-pyproject",
-
-    "mypy",
-    "boto3-stubs",
-    "django-stubs",
-    "djangorestframework-stubs",
-    "types-babel",
-    "types-bleach",
-    "types-freezegun",
-    "types-Markdown",
-    "types-retry",
-    "types-Pillow",
-    "types-simplejson",
-    "types-stripe",
-]
+description = ""
+authors = ["f213 <f@f213.in>"]
```

Если нам критично указывать автора, лучше использовать мою основную почту
```suggestion
authors = ["Fedor Borshev <fedor@borshev.com>"]
```